### DISCUSSION TITLE
v10 ETA

### DISCUSSION THREAD

**Discussion by BerserkeR-Git** [wg-easy/wg-easy#723] [General]
Just curious if v10 is coming to `release` soon, so I can do the setup of a bunch of things for a friend,.  all at once.

**Comment by pheiduck** [comment]
We have voted internally that we plan to release v10 on 2024-01-10.
https://github.com/wg-easy/wg-easy/pull/691#issue-2053043617

**Reply by BerserkeR-Git** [reply]
Understood and thank you for the update, and also all the help you've provided before.
